.github/workflows/stale.yml

```diff
@@ -8,10 +8,15 @@ jobs:
   stale:
     if: github.repository == 'PecanProject/pecan'
 
+    permissions:
```

Comment:
For my reference: Is a permissions section needed in every workflow now?

Reply:
no, but it is needed here for teh cache, otherwise it keeps processing the same PR/Issues.